packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1857,8 +1857,8 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
                     action={
                       viewRecord ||
                       isDuplicate ||
-                      declaration.registrationStatus ===
-                        SUBMISSION_STATUS.CORRECTION_REQUESTED ? null : (
```

Comment:
@naftis This is an example of the type fields being compared

Reply:
I left a couple of comments regarding this, see if they help